zarr/store/zipfile.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+
+
+from zarr.store.base import ArrayStore
+
+
+class ZipFileStore(ArrayStore):
+
+    def __init__(self, path):
+        self._path = path
+        # TODO
+
+    @property
+    def meta(self):
+        pass
+
+    @property
+    def data(self):
+        pass
+
+    @property
+    def attrs(self):
+        pass
+
+    @property
+    def cbytes(self):
+        pass
+
+    @property
+    def initialized(self):
+        pass
```

Comment:
For me, ideally, zarr wouldn't need to know anything about zipfiles and we could reduce the entire interface down to just `MutableMapping`.  That way you wouldn't need to maintain or test anything about zip, s3, hdfs, etc. and that can all be handled by other libraries.